doc/geany.txt

```diff
@@ -2068,14 +2074,13 @@ Next to current
     Whether to place file tabs next to the current tab
     rather than at the edges of the notebook.
 
-Double-clicking hides all additional widgets
-    Whether to call the View->Toggle All Additional Widgets command
-    when double-clicking on a notebook tab.
-
```

Comment:
Why removing this?  The pref itself is not documented anywhere else, merely referenced "the prefs has to be on".

Reply:
That was a mistake, thanks - restored.